src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/SampleTest.kt

```diff
@@ -0,0 +1,10 @@
+package uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi
```

Comment:
Is this still needed?

Reply:
Yes, it's just to fix gradlew test failure.